### PR TITLE
chore: update issue templates

### PR DESCRIPTION
**Pull request by H4M5TER**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿blank_issues_enabled: false
+contact_links:
+- name: Discussion
+  url: https://github.com/koishijs/koishi/discussions
+  about: Asking for help here | 在这里寻求别人的帮助
+- name: Koishi Forum
+  url: https://forum.koishi.xyz
+  about: Asking for help here | 在这里寻求别人的帮助
```
Comment (MaikoTan) [inline .github/ISSUE_TEMPLATE/config.yaml:8]: I think you could say "Koishi Official Forum" here.

File: .github/ISSUE_TEMPLATE/config.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+- name: Others
+  url: https://koishi.chat/about/contact.html
+  about: Other contacting platforms | 其他讨论方式
```
Comment (MaikoTan) [inline .github/ISSUE_TEMPLATE/config.yaml:11]:
```suggestion
  about: More Koishi Communities | 其他讨论社区
```

File: .github/ISSUE_TEMPLATE/config.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿blank_issues_enabled: false
+contact_links:
+- name: Github Discussion
+  url: https://github.com/koishijs/koishi/discussions
+  about: Asking for help here | 在这里寻求别人的帮助
+- name: Koishi Official Forum
+  url: https://forum.koishi.xyz
+  about: Asking for help here | 在这里寻求别人的帮助
+- name: Others
```
Comment (shigma) [inline .github/ISSUE_TEMPLATE/config.yaml:9]:
```suggestion
- name: More
```

More 是不是更好？

File: .github/ISSUE_TEMPLATE/config.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+﻿blank_issues_enabled: false
+contact_links:
+- name: Github Discussion
+  url: https://github.com/koishijs/koishi/discussions
+  about: Asking for help here | 在这里寻求别人的帮助
```
Comment (shigma) [inline .github/ISSUE_TEMPLATE/config.yaml:5]:
```suggestion
  about: Asking for help here | 在讨论区寻求别人的帮助
```

File: .github/ISSUE_TEMPLATE/config.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+- name: Koishi Official Forum
+  url: https://forum.koishi.xyz
+  about: Asking for help here | 在这里寻求别人的帮助
```
Comment (shigma) [inline .github/ISSUE_TEMPLATE/config.yaml:8]:
```suggestion
  about: Asking for help here | 在官方论坛寻求别人的帮助
```

File: .github/ISSUE_TEMPLATE/config.yaml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+﻿blank_issues_enabled: false
+contact_links:
+- name: Github Discussion
+  url: https://github.com/koishijs/koishi/discussions
+  about: Asking for help here | 在这里寻求别人的帮助
+- name: Koishi Official Forum
```
Comment (shigma) [inline .github/ISSUE_TEMPLATE/config.yaml:6]:
```suggestion
- name: Official Forum
```

File: .github/ISSUE_TEMPLATE/config.yaml [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+  url: https://forum.koishi.xyz
+  about: Asking for help here | 在这里寻求别人的帮助
+- name: Others
+  url: https://koishi.chat/about/contact.html
+  about: More Koishi communities | 其他讨论社区
```
Comment (shigma) [inline .github/ISSUE_TEMPLATE/config.yaml:11]:
```suggestion
  about: More Koishi communities | 加入官方 QQ 群、Discord 频道
```

**Review by MaikoTan — APPROVED** [review]

**Review by shigma — APPROVED** [review]

**Pull request merged**